<launch>

    <!-- Global parameters -->
    <param name="rw_baseline" value="1.30"/>
    <param name="fr_distance" value="1.765"/>
    <param name="steering_factor" value="32"/>
    <param name="lat_r" value="45.618932386592405" />
    <param name="lon_r" value="9.281178887031235" />
    <param name="alt_r" value="229.04906147731415" />


    <param name="use_sim_time" value="true" />


    <!-- First odometry node -->
    <node pkg="first_project" type="odometry" name="odometry" output="screen" />
    <!-- Second odometry node -->
    <node pkg="first_project" type="gps_odometer" name="gps_odometer" output="screen" />
    <!-- Third node -->
    <node pkg="first_project" type="sector_times" name="sector_times" output="screen" />

    <!-- Rviz visualization node -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find first_project)/rviz/config_file.rviz" />

    <!-- Topic echo-->
    <arg name="sector_times"     default="sector_times"/>
    <node pkg="rostopic" type="rostopic" name="topic_echo" args="echo $(arg sector_times)" output="screen"/>

</launch>
